web/src/lightbox.js

```diff
@@ -497,7 +504,7 @@ export function parse_media_data(media) {
         url: util.is_valid_url(url) ? url : "",
     };
 
-    asset_map.set(preview_src, payload);
+    asset_map.set([preview_src, $parent.attr("aria-label") || $parent.attr("href")], payload);
```

Comment:
Can this use `get_asset_map_key_for_media` too?

Reply:
@timabbott , yes you're right it can, I didn't realize it. I tested it out... works the same and am force pushing the updated one.

Reply:
@timabbott , we can't do that because ```get_asset_map_key_for_media``` returns you the metadata corresponding to the key (array of ```preview_src```  and ```title```). When you use this function inside ```asset_map.set()```, then you are trying to access a key which doesn't exist => undefined.
Hence, it fails.